src/main/resources/jshint-shell.js

```diff
@@ -28,6 +28,13 @@
     var OPTIONS_ARG = 4;
 
     var options = JSON.parse(stripJsonComments(args[OPTIONS_ARG]));
+    var globals;
+
+    if (options.globals) {
+        globals = options.globals;
+        delete options.globals;
```

Comment:
Thanks for the explanation. However assign one object to another will not cause a copy - it is just assigning a reference, correct? So I stll don't understand the deletion... sorry if this is something obvious.